index.html

```diff
@@ -8,6 +8,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
+    <script type="module" src="/src/app/index.tsx"></script>
```

Comment:
you don't need to change it

Reply:
Yeap. Moved it only for to be clear and less files